tests/functional/cli/test_cli.py

```diff
@@ -354,3 +356,52 @@ def test_can_provide_lambda_name_for_logs(runner, mock_cli_factory):
     mock_cli_factory.create_log_retriever.assert_called_with(
         mock.sentinel.Session, 'arn:aws:lambda::app-dev-foo'
     )
+
+
+def test_can_call_invoke(runner, mock_cli_factory):
+    invoke_handler = mock.Mock(spec=LambdaInvokeHandler)
+    mock_cli_factory.create_lambda_invoke_handler.return_value = invoke_handler
+    mock_cli_factory.create_payload_from_stdin.return_value = 'barbaz'
+    with runner.isolated_filesystem():
+        cli.create_new_project_skeleton('testproject')
+        os.chdir('testproject')
+        result = _run_cli_command(runner, cli.invoke, ['-n', 'foo'],
+                                  cli_factory=mock_cli_factory)
+        assert result.exit_code == 0
+    assert invoke_handler.invoke.call_args == mock.call('barbaz')
```

Comment:
I wonder if it makes sense to assert the output here in the functional tests. Not sure how feasible it is, but it seems to worthwhile to look at more than just the return code for these tests.